zenodo/modules/deposit/indexer.py

```diff
@@ -98,6 +99,15 @@ def indexer_receiver(sender, json=None, record=None, index=None,
         if relations:
             json['relations'] = relations
 
+    try:
+        for index, location in enumerate(json['locations']):
+            lat = location['lat']
+            lon = location['lon']
+            json['locations'][index]['point'] = {'lat': lat, 'lon': lon}
+
+    except:
```

Comment:
Be very careful with ``except:`` it catches everything including keyboard interrupts. Unless you have very specific reasons for using ``except:`` you should always use ``except Exception:`` instead if you need a generic handle. However, in this case, you should likely go with catching a ``KeyError``.

Reply:
Thanks, indeed I modified it using ``if json.get('location'):`` instead since the field is not required.